docs/source/inference.rst

```diff
@@ -0,0 +1,206 @@
+Inference Using Pretrained Embedding
+--------------------------------------
+
+Users can use DGL-KE to do inference tasks based on pretained embeddings (We recommand using DGL-KE to generate these embedding). Here we support two kinds of inference tasks:
+
+  * **Linkage score ranking** Given a list of (h, r, t) triplets, calculate the linkage score using the predefined score function for each triplet, sort the resulting scores and output the topk most confident triplets.
```

Comment:
i feel the description of the task isn't very intuitive.  How about this?
predicting missing entities/relations in a triplet: given entities and/or relations, predict which entities or relations are likely to connect with the existing entities for given relations. For example, given a head entity and a relation, predict which entities are likely to connect to the head entity via the given relation.

Reply:
This version is better.